qrcodereaderview/src/main/java/com/google/zxing/client/android/camera/CameraManager.java

```diff
@@ -183,7 +188,8 @@ public synchronized void startPreview() {
     if (theCamera != null && !previewing) {
       theCamera.getCamera().startPreview();
       previewing = true;
-      autoFocusManager = new AutoFocusManager(context, theCamera.getCamera());
+      autoFocusManager = new AutoFocusManager(theCamera.getCamera());
```

Comment:
This is how original Zxing project is doing it. It seems like you need to create a new AutoFocusManager everytime you change the configuration.